aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,26 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## June 4, 2022 - July 8, 2022
+
+### Stay Current with the Latest Changes
+
+- **Tabs**
+  - We've enhanced responsive behavior and updated the guidance.
+- **Menu**
```

Comment:
also add a link here
https://design-system.hpe.design/components/menu

another note is it just me or on this page are the menus automatically open? is that the behavior we want Ive been out of the loop for a little bit so might of missed that

Reply:
done